### ISSUE TITLE
Mobile Bottom drawer is not setting aria attributes

### ISSUE THREAD

**Issue by JammingBen** [opencloud-eu/web#2003]
### Initial Checklist

- [x] I understand this is a bug report and questions should be posted in the [Community Discussions](https://github.com/orgs/opencloud-eu/discussions/)
- [x] I searched [issues](https://github.com/opencloud-eu/web/issues?q=is%3Aissue) and couldn’t find anything (or linked relevant results below)

### Bug Description

Our mobile bottom drawer is not setting the expected `aria-haspopup` and `aria-expanded` attributes on the anchor element.

### Reproduction Steps

- Switch to mobile view
- Inspect any button that triggers the bottom drawer
- Open the bottom drawer

### Expected Outcome

The anchor element has `aria-haspopup` and `aria-expanded` set accordingly.

### Actual Outcome

The aria attributes are missing.
